apps/health_campaign_field_worker_app/lib/data/repositories/sync/sync_down.dart

```diff
@@ -219,7 +219,7 @@ class PerformSyncDown {
             }
 
             break;
-        // INFO : Need to add Router of package Here
+          // INFO : Need to add Router of package Here
```

Comment:
Change this to appropriate Note. It's not router here.

Reply:
> Change this to appropriate Note. It's not router here.

did the change